cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 2/11/2025 (PENDING)_
 
+**Bugfixes:**
+
+- Actions performed in `after` hooks, like `.click()` and `.type()` will now correctly retry and perform the action when a test fails. Fixes [#2831](https://github.com/cypress-io/cypress/issues/2831).
```

Comment:
This is really "all commands retry as intended" including action commands.

Reply:
I'll update this.